runtime/compiler/optimizer/InlinerTempForJ9.cpp

```diff
@@ -5106,6 +5106,23 @@ TR_InlinerFailureReason
          break;
    }
 
+#if JAVA_SPEC_VERSION >= 20
+   /**
+    * Prevent inlining of *.runWith methods because Thread.findScopedValueBindings implementation
+    * relies on iterating the O-slots of these methods to find the most recent ScopedValue binding.
+    * The scoped value binding is the first argument in runWith, and if runWith is compiled, the
+    * O-slot walker would rely on iterating through the O-slots until an object that is an instance
+    * of jdk.incubator.concurrent.ScopedValue$Snapshot is found. If we were to allow inlining of
```

Comment:
Should this be an incubator package name here?  For JDK21 I imagine it isn't.

Reply:
It's not in incubator package in JDK21, so I fixed the comment.